daemon/oci_windows.go

```diff
@@ -25,11 +25,30 @@ func (daemon *Daemon) createSpec(c *container.Container) (*specs.Spec, error) {
 	// In base spec
 	s.Hostname = c.FullHostname()
 
+	if err := daemon.setupSecretDir(c); err != nil {
```

Comment:
setupSecretDir is doing a mount/unmount, and then that's being done in the loop at line 38 immediately after. Mount/Unmount is a very expensive (relatively) operation compared to Linux (we had strong feedback from the perf team when I was doing an accidental mount/unmounts a while back and had to refactor), so I'd really like to see it limited to at most 1 mount/unmounts pair to support secrets.